jetty-server/src/test/java/org/eclipse/jetty/server/ssl/SniSslConnectionFactoryTest.java

```diff
@@ -85,11 +84,11 @@ public void before() throws Exception
         http_config.setSecureScheme("https");
         http_config.setSecurePort(8443);
         http_config.setOutputBufferSize(32768);
-        _https_config = new HttpConfiguration(http_config);
+        _httpsConfig = new HttpConfiguration(http_config);
```

Comment:
If you are getting rid of `snake_case`, then complete the effort (like `http_config` above)